src/commands/logs.ts

```diff
@@ -0,0 +1,354 @@
+  }
+
+  return lines
+    .map((line) => {
+      if (line.includes('[ERROR]')) {
+        return chalk.red(line);
+      } else if (line.includes('[WARN]')) {
+        return chalk.yellow(line);
+      } else if (line.includes('[DEBUG]')) {
+        return chalk.cyan(line);
+      } else if (line.includes('[INFO]')) {
+        return chalk.white(line);
+      }
+      return line;
+    })
+    .join('\n');
+}
+
+/**
+ * Prints a header with file information
+ */
+function printLogHeader(logPath: string, isCurrentSession: boolean): void {
+  const stats = fs.statSync(logPath);
+
+  printBorder();
+  logger.info(chalk.bold(path.basename(logPath)));
+  logger.info(chalk.gray(`Path: ${logPath}`));
+  logger.info(chalk.gray(`Size: ${formatFileSize(stats.size)}`));
+  logger.info(chalk.gray(`Modified: ${stats.mtime.toLocaleString()}`));
+
+  if (isCurrentSession) {
+    logger.info(chalk.yellow('Note: This is the log file for the current CLI session.'));
+  }
+
+  printBorder();
+}
+
+interface PrintOptions {
+  lines?: number;
+  head?: number;
+  noColor: boolean;
+  noHeader?: boolean;
+}
+
+/**
+ * Prints log file content to console with optional filtering
+ */
+async function printLogContent(logPath: string, options: PrintOptions): Promise<void> {
+  const stats = fs.statSync(logPath);
+
+  if (stats.size === 0) {
+    logger.info(chalk.gray('Log file is empty.'));
+    return;
+  }
+
+  // Warn about large files
+  const ONE_MB = 1024 * 1024;
+  if (stats.size > ONE_MB && !options.lines && !options.head) {
+    logger.warn(dedent`
+      Log file is large (${formatFileSize(stats.size)}).
+      Consider using ${chalk.cyan('-n <lines>')} to limit output.
+    `);
+  }
+
+  const content = fs.readFileSync(logPath, 'utf-8');
+  let lines = content.split('\n');
+
+  // Remove trailing empty line if present
+  if (lines[lines.length - 1] === '') {
+    lines.pop();
+  }
+
+  if (options.head) {
+    lines = lines.slice(0, options.head);
+  } else if (options.lines) {
+    lines = lines.slice(-options.lines);
+  }
+
+  const output = highlightLogLines(lines, options.noColor);
+  logger.info(output);
+}
+
+/**
+ * Follows a log file in real-time (like tail -f)
+ */
+async function followLogFile(logPath: string, noColor: boolean): Promise<void> {
+  let position = fs.existsSync(logPath) ? fs.statSync(logPath).size : 0;
+
+  logger.info(chalk.gray(`Following ${path.basename(logPath)}... (Ctrl+C to stop)\n`));
+
+  // Print existing content first (last 20 lines)
+  if (position > 0) {
+    const content = fs.readFileSync(logPath, 'utf-8');
+    const lines = content.split('\n').filter((line) => line.length > 0);
+    const lastLines = lines.slice(-20);
+    if (lastLines.length > 0) {
+      logger.info(highlightLogLines(lastLines, noColor));
+    }
+    position = fs.statSync(logPath).size;
+  }
+
+  const watcher = fs.watch(logPath, (eventType) => {
+    if (eventType === 'change') {
+      try {
+        const newSize = fs.statSync(logPath).size;
```

Comment:
I have no idea how frequently this gets fired.  Should we add some sort of debounce protection?